UserListScreen.tsx

```diff
@@ -1,15 +1,135 @@
 import React from 'react'
 import { Component } from 'react';
-import { Text } from 'react-native';
+import { Text, View, Image, FlatList, StyleSheet, ActivityIndicator, AppRegistry } from 'react-native';
+import { ListItem } from 'react-native-elements';
+
+import { ApolloClient } from 'apollo-client';
+import { ApolloProvider, Query } from 'react-apollo';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { createHttpLink } from 'apollo-link-http';
+import { setContext } from 'apollo-link-context';
+import gql from "graphql-tag";
+
+import ApolloApp from './App';
+
+import { AsyncStorage } from 'react-native';
+
+const httpLink = createHttpLink({
+  uri: "https://tq-template-server-sample.herokuapp.com/graphql"
+});
+
+const authLink = setContext(async (_, { headers }) => {
+  const token = await AsyncStorage.getItem('@Token:key');
+  return {
+    headers: {
+      ...headers,
+      authorization: token ? `${token}` : "",
+    }
+  }
+});
+
+const client = new ApolloClient({
+  link: authLink.concat(httpLink),
+  cache: new InMemoryCache()
+});
+AppRegistry.registerComponent('UserList', () => ApolloApp);
 
 export default class UserList extends Component {
     static navigationOptions = {
-        title : 'Lista do Usuário'
+        title : 'Usuários Cadastrados'
+    }
+
+    keyExtractor = (item, index) => index.toString();
```

Comment:
Não é uma boa usar index como key, ainda mais quando temos um id único por usuário do servidor. [Docs do React](https://reactjs.org/docs/lists-and-keys.html#keys) sobre isso.

Reply:
Peguei o id na query e usei como key na função `keyExtractor` (coloquei como propriedade da Query também).